app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -88,6 +89,13 @@ public void setContext(Context aContext) {
         }
     }
 
+    public GeckoRuntimeSettings getSettings() {
```

Comment:
Rename this to getRuntimeSettings? We use getSettings() in other places for GeckoSession.Settings